src/pages/applications.js

```diff
@@ -0,0 +1,105 @@
+import Footer from "../components/footer"
+import SEO from "../components/seo"
+
+import LiveCodersImg from "../img/livecoders-2.jpg"
+import ChatBubbleIcon from "../img/icons/comments-regular.svg"
+import HeartIcon from "../img/icons/heart-regular.svg"
+import KeyboardIcon from "../img/icons/keyboard-regular.svg"
+import UserIcon from "../img/icons/user-regular.svg"
+import TwitchIcon from "../img/icons/twitch-brands.svg"
+
+import "../css/applications.css"
+
+export default () => (
+  <React.Fragment>
+    <SEO title="Applications | The Live Coders" />
+    <div className="head">
+      <Header />
+      <div>
+        <h1>Apply to be part of the team</h1>
+
+        <a 
+          href="https://github.com/livecoders/Home/issues/new?assignees=csharpfritz&labels=application&template=application.md&title=Application+for+Membership"
+          className="btn"
+        >
+          Apply Now
+        </a>
+      </div>
+    </div>
+    <section id="criteria">
+      <div>
+        <h2>Qualifications</h2>
+        <p>We want to ensure that broadcasters on the team are makers who have made a commitment 
+          to broadcast on the Twitch platform with a positive and welcoming atmosphere. 
+          Members and prospective members:</p>
+          <ul>
+            <li>Are at least 18 years of age.</li>
+            <li>Twitch Affiliate or Partner status.</li>
+            <li>Minimum of 5 average viewers over the last 30 days.</li>
+            <li>During broadcasts, politely answer questions from viewers.</li>
+            <li>Promote a positive, inclusive, and welcoming chat room (we discourage excessive Blue Comedy/Ribaldry).</li>
+            <li>Regularly write code or make hardware with technology. Other content is cool.</li>
+            <li>Are a person, not a brand or company account.</li>
+          </ul>
+      </div>
+    </section>
+    <section id="process">
+      <div>
+        <h2>The Process</h2>
+        <p>
+          <ul>
+            <li>Hit the Apply Now button at the top of this page</li>
+            <li>Fill in the GitHub issue to the best of your ability</li>
+            <li>The Live Coders get alerted of your application</li>
+            <li>Any team member can give their opinions on your stream and community.</li>
+            <li>Taking all member opinions, the minimum criteria and looking closely at your VoDs 
+              or live streams, we review your application.</li>
+            <li>A decision is made and confirmed by at least one other Live Coder</li>
+            <li>You the applicant are told if you are successful at becoming a Full team member, 
+              an Intern member in the nurture program or rejected.</li>
+            <li>REJECTION: There have only ever been 3 Rejections in the past if the broadcaster 
+              meets the minimum criteria.</li>
+            <li>FULL TEAM MEMBERSHIP: Invited tot he Twitch Team, Full Access to the discord server, 
```

Comment:
"Invited tot he Twitch" -> "Invited to the Twitch"

Reply:
fixed